README.md

```diff
@@ -116,6 +146,92 @@ If the config is done correctly the test result should look like the screeshot b
+        },
+        {
+            "id": 12962576,
+            "node_id": "W_kwDOGBq-Vc4AxcsQ",
+            "name": "Pull Target Files from Applanga",
+            "path": ".github/workflows/applanga-pull.yml",
+            "state": "active",
+            "created_at": "2021-09-08T18:11:09.000+02:00",
+            "updated_at": "2021-09-08T19:11:18.000+02:00",
+            "url": "https://api.github.com/repos/oaks-view/github-workflow-example/actions/workflows/12962576",
+            "html_url": "https://github.com/oaks-view/github-workflow-example/blob/master/.github/workflows/applanga-pull.yml",
+            "badge_url": "https://github.com/oaks-view/github-workflow-example/workflows/Pull%20Target%20Files%20from%20Applanga/badge.svg"
+        },
+        {
+            "id": 12962577,
+            "node_id": "W_kwDOGBq-Vc4AxcsR",
+            "name": "Push Source Files to Applanga",
+            "path": ".github/workflows/applanga-push.yml",
+            "state": "active",
+            "created_at": "2021-09-08T18:11:09.000+02:00",
+            "updated_at": "2021-09-08T18:11:09.000+02:00",
+            "url": "https://api.github.com/repos/oaks-view/github-workflow-example/actions/workflows/12962577",
+            "html_url": "https://github.com/oaks-view/github-workflow-example/blob/master/.github/workflows/applanga-push.yml",
+            "badge_url": "https://github.com/oaks-view/github-workflow-example/workflows/Push%20Source%20Files%20to%20Applanga/badge.svg"
+        }
+    ]
+}
+```
+
+From the response you can find the workflows in the `workflow` field and the `id` field of the entries is the `workflow_id`. Once the workflow id is gotten then the url can be put together like so
+`https://api.github.com/repos/oaks-view/github-workflow-example/actions/workflows/24995310/dispatches` where `24995310` is the `workflow_id`.
+In case the workflow name is used instead a sample url would look like so
+`https://api.github.com/repos/oaks-view/github-workflow-example/actions/workflows/applanga-pull-branch.yml/dispatches`, where `applanga-pull-branch.yml` is the workflow file name. Please refer to the following screenshot 
+
+![](https://www.applanga.com/assets/images/docu/webhook_branch_trigger_endpoint_url.png)
+
+* **Headers** You need to set the following headers, `Authorization` and `Accept`. The `Authorization` header value is a valid github access token with repository permissions combined like so `token <Access token>`. For example if you had an access token `ghp_zWAdtUqmtFTY7qkqJS1wmuEx6t4X0SpIPv` then the Authorization header would be set like so `token ghp_zWAdtUqmtFTY7qkqJS1wmuEx6t4X0SpIPv`. Please refer to the following github documentation https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token on how to generate an access token.
```

Comment:
Here is also a URL text, instead of an URL. It seems to be the same as above, so the same questions apply. Could you please check?